### ISSUE TITLE
Verify Email/Password reset feature not working

### ISSUE THREAD

**Issue by Tamunoseimiebi** [Requarks/wiki#4978]
### Discussed in https://github.com/Requarks/wiki/discussions/4974

<div type='discussions-op-text'>

<sup>Originally posted by **Tamunoseimiebi** February  4, 2022</sup>
Hello, i have a self hosted wikijs running on docker. When a user signs up, a verification email is sent but upon clicking, it displays this error:
![Screenshot from 2022-02-04 16-16-36](https://user-images.githubusercontent.com/61978868/152554590-369cfbd3-4d70-4739-af05-e59e85c2188b.png)

Also, if an existing user tries to reset their pass word, they get the same error. Can someone please proffer a solution? Thanks.</div>

**Comment by craph** [comment]
Hello @Tamunoseimiebi, 

Do you have any error logs when this behavior occured ? or any error in the console log of your browser ?

Thank you very much.
Best regards,

**Comment by Tamunoseimiebi** [comment]
@craph yes, i got this from the console logs: W8bwB_JjJI8KmajCj6gwU:1 Failed to load resource: the server responded with a status of 500 ()
